.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: cpp
 before_install:
 - sudo apt-get update -qq
-- sudo apt-get install -qq libboost-regex-dev libyaml-cpp-dev cmake libgtest-dev
+- sudo apt-get install -qq libboost-regex-dev libyaml-cpp-dev cmake libgtest-dev libre2-dev
```

Comment:
I initially left it in because of the dependency on boost with the `boost::trim()` call, but I think it's better to just have a simple `trim()` in `StringUtils.h` instead, so I added that.